code/index.html

```diff
@@ -0,0 +1,85 @@
+    <body>
+        <header>
+            <a href="/"><img src="./image/Property 1=Variant3.png" class="logo"></a>
+            
+            <a href="/login"><button class="login-btn">로그인</button></a>
+        </header>
+
+        <main>
+            
+            <section class="banner top">
+                <div class="banner-content">
+                    <div class="banner-txt">
+                        <p>일상의 모든 물건을<br>거래해 보세요</p>
+                        <a href="/items"><button class="action-btn">구경하러 가기</button></a>
+                    </div>
+                    <img src="./image/Img_home_top.png" class="banner-img">
+                </div>
+            </section>
+
+            <section class="info-section">
+                <div class="info-card hot-item">
+                    <img src="./image/Img_home_01.png" class="info-card-img">
+                    <div class="info-card-txt">
+                        <span class="info-title">Hot item</span>
+                        <p class="big-info">인기 상품을<br> 확인해 보세요</p>
+                        <p class="small-info">가장 hot한 중고거래물품을<br> 판다 마켓에서 확인해 보세요</p>
+                    </div>
+                </div>
+                <div class="info-card search">
+                    <div class="info-card-txt">
+                        <span class="info-title">Search</span>
+                        <p class="big-info">구매를 원하는<br>상품을 검색하세요</p>
+                        <p class="small-info">구매하고 싶은 물품은 검색해서<br>쉽게 찾아보세요</p>
+                    </div>
+                    <img src="./image/Img_home_02.png" class="info-card-img">
+                </div>
+                <div class="info-card register">
+                    <img src="./image/Img_home_03.png" class="info-card-img">
+                    <div class="info-card-txt">
+                        <span class="info-title">Register</span>
+                        <p class="big-info">판매를 원하는<br>상품을 등록하세요</p>
+                        <p class="small-info">어떤 물건이든 판매하고 싶은 상품을<br>쉽게 등록하세요</p>
+                    </div>
+                </div>
+                
+            </section>
+            
+            <section class="banner bottom">
+                <div class="banner-content">
+                    <div class="banner-txt">
+                        <p>믿을 수 있는<br>판다마켓 중고거래</p>
+                    </div>
+                    <img src="./image/Img_home_bottom.png" class="banner-img">
+                </div>
+            </section>
+
+        </main>
+
+        <footer>
+            <div class="footer-content">
+                <span class="footer-company flex-row-center">codeit - 2024</span>
+                <div class="footer-policy-faq flex-row-center">
+                    <a href="/privacy">Privacy Policy</a>
+                    <a href="/faq">FAQ</a>
+                </div>
+                <div class="footer-sns flex-row-center">
+                    <a href="https://facebook.com" target="_blank"><img src="./image/ic_facebook.png"></a>
```

Comment:
image 의 alt 속성이 빠져있네요! alt 속성은 이미지가 렌더링되지 못할 때 나타날 문자열을 지정하기 위한 값으로 웹 접근성과 SEO 에 영향을 줄 수 있습니다. 웹 표준기관 WHATWG 의 대체 텍스트 제공 가이드라인 에도 다음과 같은 문구가 있습니다.

"Except where otherwise specified, the alt attribute must be specified and its value must not be empty"
달리 지정된 경우를 제외하고 alt 특성을 지정해야만 하며 해당 값은 비워 둘 수 없습니다.

alt 속성과 관련된 문서도 첨부드리니 참고해보세요 : )
https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt